README.md

```diff
@@ -68,6 +67,18 @@ workers implement the Remote Worker API.
 * [Scoot](https://github.com/twitter/scoot) (open source)
 * [Turbo Cache](https://github.com/allada/turbo-cache) (open source)
 
+### Workers
+Servers are responsible for managing a pool of workers.
+The Remote Worker API defines a reference protocol for worker and server
+communication, although, adhering to this protocol is not a requirement.
+Typically, workers are domain specific, optimized for custom scheduling
+protocols defined by the server implementation.
+
+* [Remote Worker API](https://goo.gl/oojM5H)
```

Comment:
I think it might be worth using the full (if ugly) google doc URL, in case goo.gl goes away. Also, could this be replaced with a link to the first mention of "Remote Worker API" above?